addons/hr_contract/tests/test_auto_status.py

```diff
@@ -45,14 +45,14 @@ def test_contract_enddate(self):
         self.assertEqual(self.contract.state, 'close')
 
     def test_contract_pending_visa_expire(self):
```

Comment:
If we're testing the work permit, the title should reflect that as well :)
```suggestion
    def test_contract_pending_work_permit_expire(self):
```